distribution/build.gradle

```diff
@@ -275,6 +275,10 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
         }
       }
     }
+    all {
+      resolutionStrategy.dependencySubstitution {
+        substitute module("org.apache.logging.log4j:log4j-core") using project(":libs:elasticsearch-log4j") because "patched to remove JndiLookup clas"}
```

Comment:
There is a small typo in `clas`.

Reply:
It would also be a good idea to put the `}` on a separate line for readability.

Reply:
Also, clas -> class (sorry for my OCD)

Reply:
Good catch. I was slightly... rushed :wink: